apps/test/unit/lib/kits/maker/PlaygroundComponentsTest.js

```diff
@@ -562,15 +615,13 @@ describe('Circuit Playground Components', () => {
     });
 
     it('destroys everything that createCircuitPlaygroundComponents creates', () => {
-      // TODO (captouch): Add 8 when re-enabled
```

Comment:
Maybe I'll just remove the one up there - these are tests that will break when we enable captouch by default, so I really shouldn't need a TODO to remind me.